packages/cover-traffic-daemon/Dockerfile

```diff
@@ -1,56 +1,60 @@
-# Run hopr-cover-traffic-daemon within a single container using npm
+ARG HOPR_TOOLCHAIN_IMAGE=${HOPR_TOOLCHAIN_IMAGE:-hopr-toolchain}
```

Comment:
The default is not a fully qualified image name and would not work, unless you have exactly that image locally available already.

Reply:
changed to default from as stated in `build-docker.sh` - `hopr-toolchain:latest`